libraries/Microsoft.Bot.Builder.Dialogs/IAdaptiveDialogDependencies.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Bot.Builder.Dialogs
+{
+    /// <summary>
+    /// Defines Dialog Dependencies interface for enumerating child dialogs should exposed for parent.
+    /// </summary>
+    public interface IAdaptiveDialogDependencies
```

Comment:
This interface should be in the Microsoft.Bot.Builder.Dialogs.Adaptive assembly, not here. Could you move it there?

Once it's there, then this interface can be, for the moment, thought of as an internal interface for the internal algorithms of replacing dialogs for the time being while it bakes.

To that end, we'd need to mark this interface internal once it's moved to `Microsoft.Bot.Builder.Dialogs.Adaptive`.


```suggestion
    internal interface IAdaptiveDialogDependencies
```